doc/source/reference/commands/tokenize.rst

```diff
@@ -212,6 +213,35 @@ See :doc:`/reference/token_filters` about token filters.
 
 .. _tokenize-return-value:
 
+``output_style``
+""""""""""""""""
+
```

Comment:
Could you write `versionadded`?

Like this: https://github.com/groonga/groonga/blob/10874e229c72a1ebeb0193c0c4b8911e73fce5bb/doc/source/reference/commands/index_column_diff.rst?plain=1#L89

Reply:
fix: [69c2f3d](https://github.com/groonga/groonga/pull/2311/commits/69c2f3da4ba2eb4a6899238e400644e060baeb40) Thank you so much. I added the next release version. I used 15.0.7 because we want to use the same version with Mroonga.
ref: https://github.com/mroonga/mroonga/releases